lib/pengine/tests/rules/pe_cron_range_satisfied_test.c

```diff
@@ -51,98 +51,130 @@ time_satisfies_year_spec(void **state) {
 static void
 time_after_year_spec(void **state) {
     run_one_test("2020-01-01",
-                 "<date_spec " PCMK_XA_ID "='spec' years='2019'/>",
+                 "<" PCMK_XE_DATE_SPEC " "
```

Comment:
Just want to call your attention to this last commit. I could drop it and we could use the literals, since I'm not sure that we have implicit tests of them elsewhere. These are user inputs via rules in the configuration.

I went with "make sure `pe_cron_range_satisfied()` works correctly given some values of the constants."

Reply:
Yeah in this case it's the first argument that's the arbitrary user input, and we're ensuring the function returns the desired value when given a well-formed date_spec